### ISSUE TITLE
DCGAN BatchNorm initialization weight looks different

### ISSUE THREAD

**Issue by DAA233** [pytorch/examples#578]
Hi there,

I used the `torch.utils.tensorboard` to watch the weight/grad when training the DCGAN example on MNIST dataset.

In the DCGAN example, we use the normal distribution to initialize both the weight of Conv and BatchNorm. However, I find it is strange when I visualize the weight of them. In the following figure, it seems that the `G/main/1/weight` (BatchNorm) is not initialized with the normal distribution because it looks so different from `G/main/0/weight` (ConvTranspose2d). It has been trained for 10 iters with batch size 64.

Could someone explain this?

![image](https://user-images.githubusercontent.com/15101533/59962209-e677b480-9514-11e9-8387-827d58fbd791.png)

The related tensorboard code is copied from [here](https://github.com/yunjey/pytorch-tutorial/blob/master/tutorials/04-utils/tensorboard/main.py):
```python
# logging weight and grads
# logging weight and grads
for tag, value in netD.named_parameters():
    tag = 'D/' + tag.replace('.', '/')
    writer.add_histogram(tag, value.data.cpu().numpy(), global_step)
    writer.add_histogram(tag+'/grad', value.grad.data.cpu().numpy(), global_step)
for tag, value in netG.named_parameters():
    tag = 'G/' + tag.replace('.', '/')
    writer.add_histogram(tag, value.data.cpu().numpy(), global_step)
    writer.add_histogram(tag+'/grad', value.grad.data.cpu().numpy(), global_step)
```
